### ISSUE TITLE
508-defect-4 [COGNITION]: VAOS Calendar - screen reader instructions should be moved to after the Next button in calendar header

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#24633]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

As a screen reader user, I would like to hear the instructions after I've heard the main calendar controls (Previous, current month, Next) so I don't lose context of what month I'm scheduling for.

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

The VA 508 office recommended we move the screen reader text after the Next button so the button's intent is kept in context. Screenshot below.

## Acceptance Criteria

- [ ] Screen reader users can hear the Previous button, month name, Next button, then SR-only instructions read in this correct order

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-appointment/request-date
* Test user Judy Morrison

## WCAG or Vendor Guidance (optional)

![Screen Shot 2021-05-12 at 4 11 22 PM](https://user-images.githubusercontent.com/934879/118045400-77401f00-b33d-11eb-9d56-daaeb7c3922b.png)

**Comment by laurenernest** [comment]
@jbalboni Will both the request and direct schedule calendar be updated with this one change or do we need another ticket for the DS calendar?
